lib/cdo/i18n_backend.rb

```diff
@@ -277,4 +277,6 @@ def eager_load!
   end
 end
 
-CDO_I18N_BACKEND = Cdo::I18n::SimpleBackend.new
+CDO_I18N_BACKEND = CDO.lazy_load_locales ?
```

Comment:
nit; could we do something like `CDO.with_default(false).lazy_load_locales` here, just for a bit of added safety?